connexion/apis/quart_api.py

```diff
@@ -0,0 +1,327 @@
+import asyncio
+import logging
+import warnings
+
+import hypercorn
+import quart
+
+from .abstract import AbstractAPI
+from connexion.apis import flask_utils
+from connexion.handlers import AuthErrorHandler
+from connexion.jsonifier import Jsonifier
+from connexion.lifecycle import ConnexionRequest, ConnexionResponse
+from connexion.security import QuartSecurityHandlerFactory
+from connexion.utils import is_json_mimetype, yamldumper
+
+logger = logging.getLogger('connexion.apis.quart_api')
+
+
+class QuartApi(AbstractAPI):
```

Comment:
There are a couple of methods in here that are identical to the FlaskApi. You might want to extract those into a super class, or subclass from the FlaskAPI.

If there are changes, it's often just a difference between using `flask` or `quart`, so you could make the framework a property of the class that you can override.

Reply:
This is also the case for the other classes in `quart_api.py` and `quart_app.py`.

Reply:
I had a go at this but couldn't immediately get it to work, probably I'm trying to do it in the wrong way.

I added a new `FlaskLikeApi` class with a class atribute `framework = None`.
Then, `FlaskApi` inherits from `FlaskLikeApi` and the framework is set to `flask`, but this raised an error in the parent class because for example a method uses `self.framework.json`, giving an AttributeError.

Reply:
Hard to say without looking at the code. But we can continue like this for now and try to refactor once everything works.